apps/avifenc.c

```diff
@@ -133,6 +133,7 @@ static void syntaxLong(void)
     printf("    --clap WN,WD,HN,HD,HON,HOD,VON,VOD: Add clap property (clean aperture). Width, Height, HOffset, VOffset (in num/denom pairs)\n");
     printf("    --irot ANGLE                      : Add irot property (rotation). [0-3], makes (90 * ANGLE) degree rotation anti-clockwise\n");
     printf("    --imir MODE                       : Add imir property (mirroring). 0=top-to-bottom, 1=left-to-right\n");
+    printf("    --clli C(,A)                      : Add clli property (content light level information). C = MaxCLL, A = MaxPALL\n");
```

Comment:
Suggestion:
```c
    printf(" --clli MAXCLL,MAXPALL : Add clli property (content light level information).\n");
```
This option is not that common so there is no need for an optional maxPALL setting. It is simpler to always request two arguments.